app/javascript/react/components/Artists/NewArtistForm.js

```diff
@@ -23,20 +23,22 @@ const NewArtistForm = () => {
     try {
       const response = await fetch("/api/v1/artists", {
         method: "POST",
+        credentials: "same-origin",
         headers: {
           "Content-Type": "application/json",
+          "Accept": "application/json",
         },
         body: JSON.stringify(newArtist),
       });
-      const artistObject = await response.json();
+      setArtistObject(await response.json());
```

Comment:
let's break this into 2 lines so that we await the response first, then set the object variable in state (that way if the await fails and throws an error we don't also try to set state!)